### ISSUE TITLE
Time the text stays on the screen for

### ISSUE THREAD

**Issue by StrangePC** [JohnCoates/Aerial#602]
I would like the option to set a custom amount of time the text stays on the screen for, not just 10 seconds or always

**Comment by NightRaider73** [comment]
Also @glouel Any way to change it from 60 mins to 1 hour?
![screen shot 2018-10-29 at 4 37 18 pm](https://user-images.githubusercontent.com/44180668/47686551-03800180-db99-11e8-8b87-e7eaa977dfcb.png)
In Brightness

**Comment by glouel** [comment]
> I would like the option to set a custom amount of time the text stays on the screen for, not just 10 seconds or always

Let me guess, you want a 5 second option that would look like what's on AppleTV ? 😅

I can see adding a 5 second option. Custom, I'm uneasy about because of the way descriptions work. A video may have several descriptions, spaced by as little as 10 or 15 seconds. That's why I use 10. Adding a custom option where you could set 30 would make little sense as it wouldn't be enforced most of the time.

**Comment by glouel** [comment]
@StrangePC ?

**Comment by NightRaider73** [comment]
@StrangePC Can you please respond?

**Comment by NightRaider73** [comment]
> Custom, I'm uneasy about because of the way descriptions work. A video may have several descriptions, spaced by as little as 10 or 15 seconds. That's why I use 10. Adding a custom option where you could set 30 would make little sense as it wouldn't be enforced most of the time.

@glouel I agree but I think knowing how much StrangePC likes it to be EXACTLY IDENTICAL maybe and a 5 second option.
